### ISSUE TITLE
Info result .log SSDT

### ISSUE THREAD

**Issue by fabio1971** [Piker-Alpha/ssdtPRGen.sh#110]
Hello Pike
Okay here.

But the kext is working?
it is loaded from the terminal
 22    0 0xffffff7f8288b000 0x12000    0x12000    com.pikeralpha.driver.AppleIntelInfo (1) <7 5 4 3 1>

**Comment by Piker-Alpha** [comment]
Yes. AppleIntelInfo.kext* is working. It shows me that Core C-States are triggered, which is good, but Package C-States not. Which is not good.

* https://github.com/Piker-Alpha/AppleIntelInfo/issues/5

**Comment by fabio1971** [comment]
Hello Pike
advice. Thanks

**Comment by Piker-Alpha** [comment]
I asked you to replace the plist, but you never replied with a different value other than 0x1 for this MSR
```
AICPUPMI: IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x1
```
So again. What value do you have now? Is that still 0x1or something else? If that is still 0x1 then please try a MacBookPro11,3 SMBIOS/board-id/model and see if that helps.

**Comment by fabio1971** [comment]
Hello Pike
I did some tests
MacBookPro 11.3 smbios
IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x5
AICPUPMI: MSR_PKG_C3_RESIDENCY.......(0x3f8) : 0x0
AICPUPMI: MSR_PKG_C6_RESIDENCY.......(0x3f9) : 0x0
AICPUPMI: MSR_PKG_C7_RESIDENCY.......(0x3fa) : 0x0
------------------------------------------------------------------------------
MacBookPro 11.3 smbios + replace the name of the file Mac-27ADBB7B4CEE8E61.plist in Mac-2BD1B31983FE1663.plist=
IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x5
AICPUPMI: MSR_PKG_C3_RESIDENCY.......(0x3f8) : 0x0
AICPUPMI: MSR_PKG_C6_RESIDENCY.......(0x3f9) : 0x0
AICPUPMI: MSR_PKG_C7_RESIDENCY.......(0x3fa) : 0x0
------------------------------------------------------------------------------
iMac 14.2 smbios + replace the name of the file Mac-27ADBB7B4CEE8E61.plist in Mac-2BD1B31983FE1663.plist=
IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x1
AICPUPMI: MSR_PKG_C3_RESIDENCY.......(0x3f8) : 0x0
AICPUPMI: MSR_PKG_C6_RESIDENCY.......(0x3f9) : 0x0
AICPUPMI: MSR_PKG_C7_RESIDENCY.......(0x3fa) : 0x0
------------------------------------------------------------------------------
iMac 14.2 
eliminated the SSDT table and added in Clover Generate C-P State
IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x1
AICPUPMI: MSR_PKG_C3_RESIDENCY.......(0x3f8) : 0x0
AICPUPMI: MSR_PKG_C6_RESIDENCY.......(0x3f9) : 0x0
AICPUPMI: MSR_PKG_C7_RESIDENCY.......(0x3fa) : 0x0
------------------------------------------------------------------------------
What can you tell me
Thanks

**Comment by fabio1971** [comment]
Hello Pike 
Have you read any advice
Thanks

**Comment by Piker-Alpha** [comment]
Hmm so even with a 5 in IA32_ENERGY_PERF_BIAS doesn't work. Is AppleLPC.kext loaded?

**Comment by fabio1971** [comment]
it is loaded from the terminal
77    0 0xffffff7f82804000 0x3000     0x3000     com.apple.driver.AppleLPC (1.7.3) <68 12 5 4 3>

**Comment by Piker-Alpha** [comment]
It won't load at boot time? That may actually be a problem. Please fix it.

**Comment by fabio1971** [comment]
injected into DSDT Device (LPCB) 0x44 0x8c 0x00 0x00

result  
AICPUPMI: IA32_ENERGY_PERF_BIAS......(0x1B0) : 0x1

AICPUPMI: MSR_PKG_C3_RESIDENCY.......(0x3f8) : 0x0
AICPUPMI: MSR_PKG_C6_RESIDENCY.......(0x3f9) : 0x0
AICPUPMI: MSR_PKG_C7_RESIDENCY.......(0x3fa) : 0x0

**Comment by fabio1971** [comment]
Hello Pike 
Have you read any advice
Thanks

**Comment by fabio1971** [comment]
Hello Pike
advice
thanks

**Comment by Piker-Alpha** [comment]
Sorry for my late response, but that is only due to me not having a clue about what is going on. The minute I figured it out, then you will be one of the first people to hear from me.
